### DISCUSSION TITLE
[Idea] Make 'Organizations' tab of user profile public

### DISCUSSION THREAD

**Discussion by adamlui** [community/community#121849] [npm]
Currently, it's only visible to the organization member so it's pretty useless (we already know if we're in the org) so why waste such beautiful code instead of unleash it upon the world?

**Comment by leminhluan4244** [comment]
Possible reasons for a private repository:
* **The code is under development and not ready for public release.** This is a common reason for keeping a repository private. The developers may want to continue working on the code and making changes before they share it with the world.
* **The code contains sensitive information.** This could include trade secrets, customer data, or personal information. Keeping the repository private helps to protect this information from unauthorized access.
* **The organization has specific plans for the code in the future.** For example, they may be planning to release it as a commercial product or open-source project. Keeping the repository private gives them more control over how and when the code is released.

**Reply by adamlui** [reply]
My idea pertains to npm not GitHub repos
